docs/resources/core/instance.md

```diff
@@ -55,7 +55,9 @@ The following arguments are supported:
 * `display_name` - (Optional) A user-friendly name. Does not have to be unique, and it's changeable. Avoid entering confidential information.
 * `image` - (Required) The OCID of the image used to boot the instance.
 * `ipxe_script` - (Optional) This is an advanced option. See the [instance API reference](https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/Instance/) for details.
-* `metadata` - (Optional) Custom metadata key/value pairs that you provide, such as the SSH public key required to connect to the instance.
+* `metadata` - (Optional) Custom metadata key/value pairs that you provide. Some possible key/value pairs:
+  * `ssh_authorized_keys` - SSH public key required to connect to the instance
```

Comment:
nit: Missing period at the end of the line for consistency.

Reply:
Fixed.